Update us_local.m3u

BellezaEmporium opened the pull request:
WSET VA

BellezaEmporium committed:
Update us_local.m3u

BellezaEmporium committed:
Update us_cbsn.m3u

Alternate

BellezaEmporium committed:
Update us_local.m3u

BellezaEmporium commented on streams/us_local.m3u:
```diff
@@ -200,6 +202,8 @@ https://cdn3.wowza.com/5/cFh0V0QwUVc4SDl2/fortlauderdale/G1118_002/playlist.m3u8
 #EXTINF:-1 tvg-id="KXVADT1.us",Fox 15 West Texas Abilene TX (KXVA)
 https://livevideo01.myfoxzone.com/hls/live/2017381/elvs/live.m3u8
+#EXTINF:-1 tvg-id="WFXTDT1.us",Fox 25 Boston (WFXT-TV) [Geo-blocked]
+https://pubads.g.doubleclick.net/ssai/event/1kItNgwtQUe54NjjBiiTGg/master.m3u8
```
https://amg00327-coxmediagroup-wfxtbreaking-ono-n5e03.amagi.tv/playlist.m3u8 GEO

BellezaEmporium committed:
Update us_local.m3u

BellezaEmporium commented on streams/us_local.m3u:
```diff
@@ -406,6 +410,8 @@ https://cdn3.wowza.com/5/WDIrTW5sM1JEY1NN/napatv/G0360_004/playlist.m3u8
 #EXTINF:-1 tvg-id="KUSADT1.us",NBC 9 Denver CO (KUSA-DT1) (1080p)
 https://livevideo.tegnadigital.com/kusa/v1/master/f9c1bf9ffd6ac86b6173a7c169ff6e3f4efbd693/KUSA/elvs/live.m3u8
+#EXTINF:-1 tvg-id="WJARDT1.us",NBC 10 Providence MA (1080p)
+https://pubads.g.doubleclick.net/ssai/event/kpannrCBQ5KwdV3Hag3Wog/master.m3u8
```
https://linear-694.frequency.stream/dist/newson/694/hls/master/playlist.m3u8

Sphinxroot reviewed (commented):
# https://www.cbsnews.com/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-cbsnews@1x.png" group-title="News",CBS News Live
https://cbsn-us.cbsnstream.cbsnews.com/out/v1/55a8648e8f134e82a470f83d562deeca/master.m3u8

# https://www.cbsnews.com/live/cbsnews-2/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-cbsnews@1x.png" group-title="News",CBS News Live 2
https://cbsn-2.cbsnstream.cbsnews.com/out/v1/a6a897e8f4f74cfc896223dfd822482f/master.m3u8

# https://www.cbsnews.com/live/cbsnews-3/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-cbsnews@1x.png" group-title="News",CBS News Live 3
https://cbsn-3-fastly.cbsnstream.cbsnews.com/out/v1/f081461c65494eeaa9ec596d0fe96a14/master.m3u8

# https://www.cbsnews.com/baltimore/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-baltimore@1x.png" group-title="News",WJZ - CBS News Baltimore
https://cbsnews.akamaized.net/hls/live/2099390/cbsnbal_11/master.m3u8

# https://www.cbsnews.com/boston/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-boston@1x.png" group-title="News",WBZ - CBS News Boston
https://cbsn-bos.cbsnstream.cbsnews.com/out/v1/589d66ec6eb8434c96c28de0370d1326/master.m3u8

# https://www.cbsnews.com/sanfrancisco/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-sanfrancisco@1x.png" group-title="News",KPIX - CBS News Bay Area
https://cbsn-sf.cbsnstream.cbsnews.com/out/v1/dac63c1abb3f4a2dac9f508f44bb072a/master.m3u8

# https://www.cbsnews.com/chicago/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-chicago@1x.png" group-title="News",WBBM - CBS News Chicago
https://cbsn-chi.cbsnstream.cbsnews.com/out/v1/b2fc0d5715d54908adf07f97d2616646/master.m3u8

# https://www.cbsnews.com/colorado/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-colorado@1x.png" group-title="",KCNC - CBS News Colorado
https://cbsn-den.cbsnstream.cbsnews.com/out/v1/2e49baf2906244ecb01b07d9885fbe7a/master.m3u8

# https://www.cbsnews.com/detroit/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-detroit@1x.png" group-title="News",WWJ - CBS News Detroit
https://cbsn-det.cbsnstream.cbsnews.com/out/v1/169f5c001bc74fa7a179b19c20fea069/master.m3u8

# https://www.cbsnews.com/losangeles/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-losangeles@1x.png" group-title="News",KCBS - CBS News Los Angeles
https://cbsn-la.cbsnstream.cbsnews.com/out/v1/57b6c4534a164accb6b1872b501e0028/master.m3u8

# https://www.cbsnews.com/miami/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-miami@1x.png" group-title="News",WFOR - CBS News Miami
https://cbsn-mia.cbsnstream.cbsnews.com/out/v1/ac174b7938264d24ae27e56f6584bca0/master.m3u8

# https://www.cbsnews.com/minnesota/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-minnesota@1x.png" group-title="News",WCCO - CBS News Minnesota
https://cbsn-min.cbsnstream.cbsnews.com/out/v1/76518f06941246ba810c8d175600bf74/master.m3u8

# https://www.cbsnews.com/newyork/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-newyork@1x.png" group-title="News",WCBS - CBS News New York
https://cbsn-ny.cbsnstream.cbsnews.com/out/v1/ec3897d58a9b45129a77d67aa247d136/master.m3u8

# https://www.cbsnews.com/philadelphia/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-philadelphia@1x.png" group-title="News",KYW - CBS News Philadelphia 
https://cbsn-phi.cbsnstream.cbsnews.com/out/v1/5c9ad3e215984b0e9ad845b335216b72/master.m3u8

# https://www.cbsnews.com/pittsburgh/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-pittsburgh@1x.png" group-title="News",KDKA - CBS News Pittsburgh
https://cbsn-pit.cbsnstream.cbsnews.com/out/v1/6966dabf8150405ab26f854e3cd6a2b8/master.m3u8

# https://www.cbsnews.com/sacramento/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-sacramento@1x.png" group-title="",KOVR - CBS News Sacramento
https://live979c117.airspace-f.cbsivideo.com/cbsnus-master/master/cbsnus-master.m3u8

# https://www.cbsnews.com/texas/live/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-texas@1x.png" group-title="News",KTVT - CBS News Texas
https://cbsn-dal.cbsnstream.cbsnews.com/out/v1/ffa98bbf7d2b4c038c229bd4d9122708/master.m3u8

# https://www.cbsnews.com/live/cbs-sports-hq/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-cbs-sports-hq@1x.png" group-title="Sports",CBS Sports HQ (GEO)
https://dai.google.com/linear/hls/event/9Lq0ERvoSR-z9AwvFS-xYA/master.m3u8

# https://www.cbsnews.com/live/inside-edition/
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-inside-edition@1x.png" group-title="",Inside Edition 24/7
https://dai.google.com/linear/hls/event/Y2DMgfclT3S2nNxANeIawA/master.m3u8

# https://www.cbsnews.com/live/mixible/ https://www.etonline.com/live/default-slug
#EXTINF:-1 tvg-logo="https://www.cbsnews.com/assets/live_cards/video-door-mixible@1x.png" group-title="",Mixible (ET LIVE) 
https://dai.google.com/linear/hls/event/xrVrJYTmTfitfXBQfeZByQ/master.m3u8

BellezaEmporium committed:
Update us_local.m3u

BellezaEmporium committed:
Update us_local.m3u

BellezaEmporium committed:
Update us_local.m3u

BellezaEmporium committed:
Update us_cbsn.m3u

BellezaEmporium committed:
Update us.m3u

BellezaEmporium commented:
Good to go for now.

Dum4G commented on streams/us.m3u:
```diff
@@ -393,6 +393,8 @@ https://ov.ottera.tv/live/master.m3u8?channel=elec_en
 #EXTINF:-1 tvg-id="EntrepreneurTV.us",Entrepreneur (720p)
 https://a.jsrdn.com/broadcast/7582ed85f7/+0000/c.m3u8
+#EXTINF:-1 tvg-id="EntertainmentTonight.us",Entertainment Tonight (720p) [Geo-blocked]
+https://etlive-mediapackage-fastly.cbsaavideo.com/dvr/manifest.m3u8
```
got 403 even from Miami

BellezaEmporium replied:
https://www.cbsnews.com/colorado/xml-sitemap-video/cbs-news-live/2023-11.xml

https://www.cbsnews.com/colorado/xml-sitemap-video/cbs-news-live/2024-01.xml

Seems to have indicated this link.

Dum4G replied:
@Alstruit @PopeyeTheSai10r

MapGuy11 replied:
I'm getting a 403 on this: https://etlive-mediapackage-fastly.cbsaavideo.com/dvr/manifest.m3u8
I think they only allow the dai.google.com to start because if you see the .ts files are being served from:
https://cbsta49f-cbsta49f-ms.global.ssl.fastly.net/amagi7b98-AmagiMixible/master/amagi7b98-AmagiMixible_6/00797/amagi7b98-AmagiMixible_6_20240213T172144_6_00022.ts

https://www.cbsnews.com/philadelphia/xml-sitemap-video/cbs-news-live/2024-01.xml

PopeyeTheSai10r replied:
Got 403 from ATL.

BellezaEmporium replied:
https://dai.google.com/linear/hls/event/xrVrJYTmTfitfXBQfeZByQ/master.m3u8 works

Dum4G commented on streams/us_local.m3u:
```diff
@@ -406,6 +408,8 @@ https://cdn3.wowza.com/5/WDIrTW5sM1JEY1NN/napatv/G0360_004/playlist.m3u8
 #EXTINF:-1 tvg-id="KUSADT1.us",NBC 9 Denver CO (KUSA-DT1) (1080p)
 https://livevideo.tegnadigital.com/kusa/v1/master/f9c1bf9ffd6ac86b6173a7c169ff6e3f4efbd693/KUSA/elvs/live.m3u8
+#EXTINF:-1 tvg-id="WJARDT1.us",NBC 10 Providence MA (1080p)
+https://pubads.g.doubleclick.net/ssai/event/kpannrCBQ5KwdV3Hag3Wog/master.m3u8
```
also geo-blocked?

Dum4G replied:
I mean it is absolutely sure geo-blocked just like the link above from the same source

BellezaEmporium replied:
Shouldn't be, it's a redirect to https://linear-694.frequency.stream/dist/newson/694/hls/master/playlist.m3u8

Dum4G replied:
Geo-blocked just for me then

PopeyeTheSai10r replied:
> Shouldn't be, it's a redirect to https://linear-694.frequency.stream/dist/newson/694/hls/master/playlist.m3u8

This one is working for me.

BellezaEmporium committed:
Update us.m3u

Change ET link
Remove "External Ottera" as it's a Toon Goggles stream that no longer works.
